lib/diagnostic.rb

```diff
@@ -5,36 +5,37 @@ class Diagnostic
   # Assume each entry in the list implements a length method that returns
   # a Fixnum.
   def longest(list)
-    # Your code goes here
+      for longest do
+        longest.length.max
```

Comment:
You need the spaceship operator since it's a comparison!

`list.max { |a, b| a.length <=> b.length }`